src/tile/MediaView.module.css

```diff
@@ -74,9 +74,9 @@ unconditionally select the container so we can use cqmin units */
     calc(var(--media-view-border-radius) - var(--cpd-space-3x))
   );
   display: grid;
-  grid-template-columns: 1fr auto;
+  grid-template-columns: 30px 1fr 30px;
```

Comment:
This should be one of the compound spaces i think?
8x most likely (32px)
```suggestion
  grid-template-columns: var(--cpd-space-8x) 1fr var(--cpd-space-8x);
```

Reply:
This value is actually linked to the size of the reaction in top left and the options button in bottom right.

<img width="887" alt="image" src="https://github.com/user-attachments/assets/3bbe3bbc-7d92-4b6b-b60f-1778c5fe3f2f">

Top left is hard coded to 30px at https://github.com/element-hq/element-call/blob/133dc26642e9e71c33634223f92fc1691d1ec15b/src/reactions/ReactionIndicator.module.css#L59

Bottom right appears to be hard coded to 20px (+ some margin) at https://github.com/element-hq/element-call/blob/133dc26642e9e71c33634223f92fc1691d1ec15b/src/tile/GridTile.tsx#L169

I'm very open to suggestions on how to properly handle this? I can do the compound spacing as suggested?

Reply:
Oh.
I see how we then want that in favor of the spacing...

Reply:
Lets not do this in here and try to remember bringing it up in a team sync.